api/postgres/postgres.go

```diff
@@ -430,17 +473,18 @@ func (u *UserProfileDB) CreateUserProfile(request *CreateUserDatabaseRequest) *C
 
 	resp := &CreateUserDatabaseResponse{Message: "Successfully added user", Error: nil}
 
-	query := "INSERT INTO profile_details (id, username, displayname) VALUES (gen_random_uuid(), $1, $2) RETURNING id, username, displayname"
+	query := `INSERT INTO profile_details (username, accountpassword, displayname) 
+			  VALUES ($1, crypt($2, gen_salt('bf',8)), $3) 
```

Comment:
This is a great salt gen implementation!!! I'm used to the 'kosher' version, but this is still excellent.